app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt

```diff
@@ -491,4 +533,5 @@ data class GlobalAppState(
     val updateAppDialog: Boolean = false,
     val conversationJoinedDialog: JoinConversationViaCodeState? = null,
     val newClientDialog: NewClientData? = null,
+    val newClientDialogRemembered: NewClientData? = null,
```

Comment:
I am still confused about what does this variable mean. Maybe some extra documentation will be helpful here.